chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -36,6 +36,9 @@
     libshadow-ruby1.8
     markdown
   )
+  unless search(:node, "platform:windows").empty?
+    pkglist.push "smbclient"
```

Comment:
http://packages.ubuntu.com/trusty/net/smbclient

Well, the package name is smbclient.